src/ActionList/Item.tsx

```diff
@@ -1,15 +1,15 @@
-import React from 'react'
 import {ForwardRefComponent as PolymorphicForwardRefComponent} from '@radix-ui/react-polymorphic'
 import {useSSRSafeId} from '@react-aria/ssr'
+import React from 'react'
 import styled from 'styled-components'
-import {useTheme} from '../ThemeProvider'
 import Box, {BoxProps} from '../Box'
-import sx, {SxProp, merge} from '../sx'
+import sx, {merge, SxProp} from '../sx'
+import {useTheme} from '../ThemeProvider'
 import createSlots from '../utils/create-slots'
 import {AriaRole} from '../utils/types'
-import {ListContext, ActionListProps} from './List'
-import {GroupContext, ActionListGroupProps} from './Group'
 import {ActionListContainerContext} from './ActionListContainerContext'
+import {ActionListGroupProps, GroupContext} from './Group'
+import {ActionListProps, ListContext} from './List'
```

Comment:
👀 just curious, do you have automatic sorting in your editor?

Reply:
😅  yup